packages/workflow-core/src/lib/plugins/external-plugin/api-plugin.ts

```diff
@@ -146,17 +155,19 @@ export class ApiPlugin {
       const isPlaceholderSecret = variableKey.includes('secret.');
       const placeholderValue = isPlaceholderSecret
         ? `${process.env[variableKey.replace('secret.', '')]}`
-        : `${this.fetchObjectPlaceholderValue(context, variableKey)}`;
+        : // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
+          `${this.fetchObjectPlaceholderValue(context, variableKey)}`;
       replacedContent = replacedContent.replace(placeholder, placeholderValue);
     });
 
     return replacedContent;
   }
 
   fetchObjectPlaceholderValue(record: AnyRecord, path: string) {
-    let pathToValue = path.split('.');
+    const pathToValue = path.split('.');
 
     return pathToValue.reduce((acc: unknown, pathKey: string) => {
+      // eslint-disable-next-line no-prototype-builtins
       if (typeof acc === 'object' && acc !== null && acc.hasOwnProperty(pathKey)) {
```

Comment:
Check out https://dev.to/aman_singh/what-s-the-deal-with-object-prototype-hasownproperty-call-4mbj